lib/rubocop/cop/style/frozen_string_literal_comment.rb

```diff
@@ -51,29 +51,64 @@ module Style
       #   module Baz
       #     # ...
       #   end
```

Comment:
Please add one blank line. This is a nitpick :-)

```diff
      #   end
+     #
      # @example EnforcedStyle: always_true
```

Reply:
@koic Good call! If there was ever a repo in which to nitpick, this is it :)

Blank line added!